onnxruntime/__init__.py

```diff
@@ -22,8 +22,8 @@
 try:
     from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, \
         RunOptions, SessionOptions, set_default_logger_severity, enable_telemetry_events, disable_telemetry_events, \
-        NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode, ExecutionOrder, OrtDevice, SessionIOBinding, \
```

Comment:
Why is OrtDevice removed ?

Reply:
It was moved below, as it is now exposed as a wrapper class